### PR TITLE
Add strong warning against production use to example init_db.sql file

### PR DESCRIPTION
**Pull request by teejae**
Signed-off-by: Toliver Jue <toliver@planetscale.com>

### CODE REVIEW

File: config/init_db.sql [hunk 1]
```diff
@@ -1,9 +1,15 @@
 # This file is executed immediately after mysql_install_db,
 # to initialize a fresh data directory.
 
-##########################################
+###############################################################################
+# WARNING: This sql is *NOT* safe for production use,
+#          as it contains default users and passwords
+#          with more privileges than necessary.
```
Comment (enisoc) [inline config/init_db.sql]: As far as I know, the privileges given are necessary. Is there an example you see that's unnecessary?

The main thing that we intended for people to change but didn't document well enough is the passwords, especially for the users that allow remote connections.

Comment (teejae) [reply config/init_db.sql]: I've removed comment about privileges, and added comment about rotating them as necessary.

**Comment by deepthi** [comment]
@teejae the second commit did not pass the DCO check, it didn't have a
signoff. Since this has already been merged I'm not sure what the best way
to fix that is, but it needs to be fixed.

**Comment by enisoc** [comment]
The single, squashed commit seems to have the DCO: https://github.com/vitessio/vitess/commit/7329452df3b39a780ff62a1fa2431e327bbbdce4

**Comment by deepthi** [comment]
I see that now. This is confusing because github is still showing 2 commits on the PR, but looks like we should be fine.

**Pull request merged**